ci/build.sh

```diff
@@ -3,5 +3,5 @@
 set -e -x
 
 cd s3-broker-app
-mvn package
+mvn package -Dmaven.test.skip=true
```

Comment:
I thought this was temporary...? We decided we did want to run the unit/integration as a prerequisite to deploy, right?